packages/ui/src/lib/components/DatePicker/DatePicker.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  import DatePickerInternal from '$lib/internal/DatePicker.svelte';
+  import type { DatePickerProps } from '$lib/types.js';
+  import { CalendarDate, type DateValue } from '@internationalized/date';
+  import { DateTime } from 'luxon';
+
+  let { value = $bindable<DateTime | null>(null), onChange, minDate, maxDate, ...rest }: DatePickerProps = $props();
+
+  // Note: DatePickerInternal/bits.ui uses DateValue, while this component uses DateTime from luxon
+
+  // Convert DateTime to DateValue
+  function toDateValue(dt: DateTime | null | undefined): DateValue | undefined {
+    if (!dt) return undefined;
+    return new CalendarDate(dt.year, dt.month, dt.day);
+  }
```

Comment:
```suggestion
  const toDateValue = (dateTime?: DateTime | null) => {
    if (!dateTime) {
      return;
    }
    return new CalendarDate(dateTime.year, dateTime.month, dateTime.day);
  }
```

(same for the ones below)

Reply:
Also, I wonder if we even need `null` here?